pandas/tests/tslibs/test_conversion.py

```diff
@@ -66,3 +66,12 @@ def test_length_zero_copy(dtype, copy):
     arr = np.array([], dtype=dtype)
     result = conversion.ensure_datetime64ns(arr, copy=copy)
     assert result.base is (None if copy else arr)
+
+
+def test_tz_convert_freq():
+    import pandas as pd
+    import pytz
```

Comment:
Don't need to import this. Just the string `'UTC'` will automatically use `pytz.UTC`